models/base/bing_ad_groups.sql

```diff
@@ -1 +1,11 @@
-{{ stitch_bing_ad_groups() }}
\ No newline at end of file
+{% if var('etl') == 'stitch' %}
+
+    {{ stitch_bing_ad_groups() }}
+    
+{% endif %}
```

Comment:
don't use two separate if/end blocks...use else :) 

`{% elif var('etl') == 'fivetran' %}`

Reply:
same comment for all of the other models as well.